cmf-cli/Handlers/PackageType/SecurityPortalPackageTypeHandler.cs

```diff
@@ -79,7 +79,7 @@ public override void Pack(IDirectoryInfo packageOutputDir, IDirectoryInfo output
 
                 FinalArchive(packageOutputDir, outputDir);
 
-                Log.Information($"{outputDir.FullName}/{CmfPackage.ZipPackageName} created");
+                Log.Debug($"{outputDir.FullName}/{CmfPackage.ZipPackageName} created");
```

Comment:
missing the Log.Information here for consistency?

Reply:
yap